Wikipedia/Code/WMFReferencePopoverMessageViewController.m

```diff
@@ -129,4 +123,29 @@ - (BOOL)accessibilityPerformEscape {
     return true;
 }
 
+- (void)applyTheme:(WMFTheme *)theme {
+    self.theme = theme;
+    if (self.viewIfLoaded == nil) {
+        return;
+    }
+
+    self.view.backgroundColor = theme.colors.paperBackground;
+    
+    self.textView.linkTextAttributes = @{NSForegroundColorAttributeName: theme.colors.link};
+    
+    [self.textView setAttributedText:[self attributedStringForHTML:[self referenceHTMLWithSurroundingHTML]]];
```

Comment:
The attributed text depends on the colors of the theme, so it needs to be updated anytime the theme changes, so it needs to be inside of `applyTheme:`